src/Tickets/Admin/All_Tickets/List_Table.php

```diff
@@ -189,49 +190,329 @@ public function get_sortable_columns() {
+			'ticket_link' => $edit_post_link,
 		];
 
-		$args = [
-			'page'               => $current_page,
-			'per_page'           => $per_page,
-			'return_total_found' => true,
-			'order'              => 'DESC',
+		return $admin_views->template( 'all-tickets/table-column-name', $context, false );
+	}
+
+	/**
+	 * Get the column ID value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_id( $item ): string {
+		return $item->ID;
+	}
+
+	/**
+	 * Get the column event value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_event( $item ): string {
+		$event = $item->get_event();
+		if ( ! $event ) {
+			return '-';
+		}
+
+		$edit_post_url  = get_edit_post_link( $event );
+		$edit_post_link = sprintf(
+			'<a href="%s" class="tec-tickets-all-tickets-table-event-link" target="_blank" rel="nofollow noopener">%s</a>',
+			esc_url( $edit_post_url ),
+			get_the_title( $event )
+		);
+
+		$orders_report_url  = add_query_arg(
+			[
+				'post_type' => $event->post_type,
+				'page'      => 'tickets-orders',
+				'event_id'  => $event->ID,
+			],
+			admin_url( 'edit.php' )
+		);
+		$orders_report_link = sprintf(
+			'<a href="%s" class="tec-tickets-all-tickets-table-event-link" target="_blank" rel="nofollow noopener">%s</a>',
+			esc_url( $orders_report_url ),
+			esc_html__( 'Orders', 'event-tickets' )
+		);
+
+		$attendees_report_url  = add_query_arg(
+			[
+				'post_type' => $event->post_type,
+				'page'      => 'tickets-attendees',
+				'event_id'  => $event->ID,
+			],
+			admin_url( 'edit.php' )
+		);
+		$attendees_report_link = sprintf(
+			'<a href="%s" class="tec-tickets-all-tickets-table-event-link" target="_blank" rel="nofollow noopener">%s</a>',
+			esc_url( $attendees_report_url ),
+			esc_html__( 'Attendees', 'event-tickets' )
+		);
+
+		$actions = [
+			'orders'    => $orders_report_link,
+			'attendees' => $attendees_report_link,
 		];
 
-		// Setup sorting args.
-		if ( tribe_get_request_var( 'orderby' ) ) {
-			$args['orderby'] = tribe_get_request_var( 'orderby' );
+		/**
+		 * Filters the actions for the event in the All Tickets Table.
+		 *
+		 * @since TBD
+		 *
+		 * @param array $actions The actions for the event in the All Tickets Table.
+		 */
+		$actions = apply_filters( 'tec_tickets_all_tickets_table_event_actions', $actions, $event, $item );
+
+		return sprintf( '%1$s %2$s', $edit_post_link, $this->row_actions( $actions ) );
+	}
+
+	/**
+	 * Get the column start date value.
+	 *
+	 * @since TBD
+	 *
+	 * @param Tribe__Tickets__Ticket_Object $item The current item.
+	 *
+	 * @return string
+	 */
+	public function column_start( $item ): string {
+		$date_format = tribe_get_date_format( true );
+		$datetime    = $item->start_date( false );
+
+		return $datetime->format( $date_format );
```

Comment:
```suggestion
		return esc_html( $datetime->format( $date_format ) );
```

Also maybe we could use the `<time>` element here ? see an example [here](https://github.com/the-events-calendar/event-tickets/blob/master/src/Tickets/Commerce/Admin_Tables/Orders_Table.php#L406)